atd-vze/src/Components/DataTable.js

```diff
@@ -46,7 +49,20 @@ const DataTable = ({
         return (
           <Col key={i} md="6">
             <Card key={section.title}>
-              <CardHeader>{section.title}</CardHeader>
+              <CardHeader>
+                <Row>
+                  <Col>{section.title}</Col>
+                  {section.button && (
```

Comment:
alternatively this condition could just be based on the `section.title` if we dont plan on any other data tables ever having buttons... and then could remove the additional `button` attribute i added to `crashDataMap.js`

Reply:
i updated this code to do the alternative option in this comment but not sure which i like more...

Reply:
oops i missed this comment before. i think what you've got is just fine. we can revisit this if we need to make other carve-outs for special features in the future.

Reply:
+1 - i had similar thoughts about an "action button" type prop or something, but I agree it is still early to know where this would go. Very cool to see it in the header!